content/en/docs/HOWTOs/EventFeeds/TranslationHowTo.md

```diff
@@ -0,0 +1,638 @@
+---
```

Comment:
I would add some words to the sending-data/_index.md to say that a common practice is for client system developers/admins to write the translations to normalise their data, then provide a link to your page.

Also rename this file to lower-kebab-case pls.  I know the other HowTos are all wrong.